interview-questions.md

```diff
@@ -8,63 +8,73 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is a function? Why would you use one?
 
   Your answer:
-
+    It is a block of code that performs a specific task
```

Comment:
I would elaborate a bit deeper here- The interviewer is also asking, why would you use one?